src/citation/views/citation_entry_view.py

```diff
@@ -163,16 +180,22 @@ def url_search(self, request):
                 parsed_url = urlparse(url_string)
                 cleaned_dois = clean_dois(parsed_url, list(map(str.strip, dois)))
                 doi_counter = Counter(cleaned_dois)
-                formmated_dois = [doi for doi, _ in doi_counter.most_common(1)]
+                formatted_dois = [doi for doi, _ in doi_counter.most_common(1)]
 
-                if len(formmated_dois) == 0:
+                if len(formatted_dois) == 0:
                     raise DOINotFoundError()
 
-                open_alex = OpenAlex()
-                open_alex_result = open_alex.get_data_from_doi(formmated_dois[0])
-
-                return Response(open_alex_result, status=200)
-        except Exception as error:
+                doi = formatted_dois[0]
```

Comment:
Are we only trying the first DOI because it is likely the `formatted_dois` will only hold a single DOI for a paper?
Maybe we add a comment in here since it is slightly unclear